scripts/upgrade_admin_server.sh

```diff
@@ -125,6 +139,8 @@ ListenStream=[::]:4369
 FreeBind=true
 EOF
 
+    systemctl start rabbitmq-server
```

Comment:
will this autostart couchdb?

Reply:
looking at the rpm scripts, couchdb should be (re)started on package upgrade.